tasks/main.yml

```diff
@@ -1,4 +1,21 @@
 ---
+- name: are we configuring kerberos?
+  set_fact:
+    aaa_sso_krb5: true
+  when: aaa_sso_keytab is defined or aaa_sso_remote_keytab is defined
+
+- name: assert we don't have aaa_sso_keytab and aaa_sso_remote_keytab together
+  assert:
+    that: aaa_sso_remote_keytab is not defined
+    msg: >
+      Variables aaa_sso_keytab and aaa_sso_remote_keytab are mutually exclusive
+  when: aaa_sso_keytab is defined
+
+- name: set up remote keytab path
+  set_fact:
+    aaa_sso_keytab_path: "{{ aaa_sso_remote_keytab }}"
+  when: aaa_sso_remote_keytab is defined
```

Comment:
I am not sure I follow this code. When you just set `aaa_sso_keytab_path` to `aaa_sso_remote_keytab` always, when both are not set, but both variables  behave the same.

Reply:
Hi, I created this variable in order to have a simple variable expansion in sso template no matter if keytab is copied from controller or already present on remote machine. This variable is set to remote keytab path when remote keytab is used, in case of copied keytab, it's set to `/etc/httpd/httpd.keytab` (specified in defaults/main.yml).